PastaGA.py

```diff
@@ -0,0 +1,354 @@
+__author__ = 'theep_000'
+
+import random
+import operator
+from collections import Counter
+import sys
+import argparse
+
+mutate_chance = 90
+base_pop = 100
+generation_max = 100
+
+poss_ingredients = ["tomato sauce","pesto","mozzarella","alfredo sauce", "pasta shells","chile sauce", "pasta bowties", "parmesan","meatballs","pasta tubes","shredded chicken","basil", "spinach","chives","shrimp","mushroom","macaroni","feta cheese","noodles","sausage","olives"]
+
+class pasta:
```

Comment:
A better name would be PastaDish. It's important to make sure you class names represent the true abstraction. This makes it seem like this class represents a single type of Pasta, when really it's a meal.